<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-210</y>
     <width>1040</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/bbb111.jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>491</width>
     <height>216</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="newName_label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 105, 214,0.6);

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>          Имя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="newName_lineEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #ec5b68;
 background-color: rgba(19,9,10, 0.6); ;
font: 700 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="newPswd_label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 105, 214,0.6);

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>          Пароль</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="newPaswd_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #ec5b68;
 background-color: rgba(19,9,10, 0.6); ;
font: 700 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="newPswd2_label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 105, 214,0.6);

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>  Повторите пароль</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="newPaswd2_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #ec5b68;
 background-color: rgba(19,9,10, 0.6); ;
font: 700 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="key_label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 105, 214,0.6);

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>  Секретная фраза</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="key_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #ec5b68;
 background-color: rgba(19,9,10, 0.6); ;
font: 700 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="info_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="create_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(163,84,143, 0.8);
color: #f3f2f8;

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
 border-radius: 55px;</string>
   </property>
   <property name="text">
    <string>сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(177,96,105, 0.8);
color: #f3f2f8;

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
 border-radius: 40px;</string>
   </property>
   <property name="text">
    <string>отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
